### PR TITLE
Install the Service Points feature

### PR DESCRIPTION
**Pull request by artem-png**
## PR Description

Ticket: https://spryker.atlassian.net/browse/CC-24931

### CODE REVIEW

**Review by kkicheglovspryker — COMMENTED** [review]
You need to wait until https://github.com/spryker/spryker-docs/pull/1905 will be merged and resolve conflicts

**Review by kkicheglovspryker — DISMISSED** [review]

File: _includes/pbc/all/install-features/202304.0/install-the-service-points-feature.md [hunk 1]
```diff
@@ -294,6 +302,23 @@ s2,sp2,pickup,1
 | service_type_key  | mandatory | string    | pickup            | Unique key of the service type.   |
 | is_active         | mandatory | bool      | 0                 | Defines if the service is active. |
 
+**data/import/common/common/marketplace/product_offer_service.csv**
+
+```csv
+service_key,product_offer_reference
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/install-features/202304.0/install-the-service-points-feature.md:308]: needs to be updated according to columns order exchange

File: _includes/pbc/all/install-features/202304.0/install-the-service-points-feature.md [hunk 2]
```diff
@@ -1033,7 +1077,65 @@ Make sure you are able to see data in Redis in the following format:
 
 ### 7) Set up behavior
 
-1. To enable the Backend API, register the plugins:
+2. To expand product offers with services, register the plugins:
```
Comment (DmitryLymarenko) [inline _includes/pbc/all/install-features/202304.0/install-the-service-points-feature.md:1080]: it looks like paragraph `### 7)` starts from `2.`
where is `1.`?

**Review by DmitryLymarenko — CHANGES_REQUESTED** [review]

**Review by DmitryLymarenko — APPROVED** [review]

**Review by andriispryker — APPROVED** [review]
Approved.

**Review by VadymSachenko — APPROVED** [review]

**Pull request merged**